### DISCUSSION TITLE
unable to rename in place of the URL base name on the data object importer's transformation pipeline.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15816] [Q&A / Support] [answered]
[AssetImportTesting.xlsx](https://github.com/pimcore/pimcore/files/12429301/AssetImportTesting.xlsx)
I need to rename the asset because the data objects have an image URL, but the image URL base name contained with the timestamps.

URL = https://cdn11.bigcommerce.com/s-99kn4fj7jr/products/134/images/396/H90PTH_1__85430.1607300804.1280.1280.jpg 
Base Name =H90PTH_1__85430.1607300804.1280.1280.jpg
Need to Rename =CS-H90PTH_T

![image](https://github.com/pimcore/pimcore/assets/106297450/a9537ce6-ed24-4a68-9d4e-6482a36efb5a)

If the data object importer has any options on the pipeline, or if I have to write a new script to do that, please let me know.

@kingjia90

**Comment by kingjia90** [comment, marked as answer]
Do you mean that  you need to rename 
`https://cdn11.bigcommerce.com/s-99kn4fj7jr/products/134/images/396/H90PTH_1__85430.1607300804.1280.1280.jpg` to 
`https://cdn11.bigcommerce.com/s-99kn4fj7jr/products/134/images/396/H90PTH_T_1__85430.1607300804.1280.1280.jpg`

Unfortunately don't know any way to do that out of the box
